tests/beaker_tests/radius_server/radius_server_provider_defaults.rb

```diff
@@ -71,6 +71,14 @@
     cmd_str = get_namespace_cmd(agent, PUPPET_BINPATH +
       'agent -t', options)
     on(agent, cmd_str, acceptable_exit_codes: [0, 2])
+
+    # Make sure radius server is not configured before test starts.
+    on(master, RadiusServerLib.create_radius_server_manifest_absent_ipv6)
+
+    # Expected exit_code is 0,2 since server may or may not be configured.
+    cmd_str = get_namespace_cmd(agent, PUPPET_BINPATH +
+      'agent -t', options)
+    on(agent, cmd_str, acceptable_exit_codes: [0, 2])
```

Comment:
This is fine but note (for future PRs) that Mike added a helper function to utilitylib.rb that makes this easy. Just call:

```
resource_absent_cleanup(agent, 'radius_server')
```